.github/workflows/build_and_test.yml

```diff
@@ -43,24 +44,24 @@ jobs:
       docker-image: ${{ needs.build.outputs.docker-image }}
       runner: linux.8xlarge.nvidia.gpu
       timeout-minutes: 180
+      collect-coverage: github.event.ref == 'refs/heads/gpu_ci_coverage'
       disable-xrt: 1
     secrets:
       gcloud-service-key: ${{ secrets.GCLOUD_SERVICE_KEY }}
 
-  test-cpu-coverage:
-    name: "Collect CPU test coverage"
-    if: github.event_name == 'push' && github.event.ref == 'refs/heads/master'
-    uses: ./.github/workflows/_coverage.yml
-    needs: build
-    with:
-      docker-image: ${{ needs.build.outputs.docker-image }}
-      collect-coverage: true
-      timeout-minutes: 120
-      disable-xrt: 1
-    secrets:
-      gcloud-service-key: ${{ secrets.GCLOUD_SERVICE_KEY }}
+  # test-cpu-coverage:
+  #   name: "Collect CPU test coverage"
+  #   if: github.event_name == 'push' && github.event.ref == 'refs/heads/master'
+  #   uses: ./.github/workflows/_coverage.yml
+  #   needs: build
+  #   with:
+  #     docker-image: ${{ needs.build.outputs.docker-image }}
+  #     collect-coverage: true
+  #     timeout-minutes: 120
+  #     disable-xrt: 1
+  #   secrets:
+  #     gcloud-service-key: ${{ secrets.GCLOUD_SERVICE_KEY }}
 
-  # TODO(yeounoh) re-enable after resolving the timeout issue
```

Comment:
do you need to uncomment the block below?

Reply:
Still testing, will remove when it's ready.